packages/node_modules/@ciscospark/internal-plugin-reaction/README.md

```diff
@@ -0,0 +1,108 @@
+# @ciscospark/internal-plugin-reaction
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Plugin for the Reaction service
+
+This is an internal Cisco Spark plugin. As such, it does not strictly adhere to semantic versioning. Use at your own risk. If you're not working on one of our first party clients, please look at our [developer api](https://developer.ciscospark.com/getting-started.html) and stick to our public plugins.
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+## Install
+
+```bash
+npm install --save @ciscospark/internal-plugin-reaction
+```
+
+## Usage
+
+
+Wiki design: https://sqbu-github.cisco.com/WebExSquared/cloud-apps/wiki/Reactions(Design-&-Api-Spec)
```

Comment:
Since this is a public repo, we'll need to remove reference to sqbu and rely on the tests to show full implementation and expectations from the service.

Reply:
yeah, I understand. I thought it would be helpful temporarily while building this out